app/views/integrated/application_submitted/edit.html.erb

```diff
@@ -76,8 +74,21 @@
         </div>
       </div>
     </div>
+    <%= form_for @feedback_form, url: feedback_path, builder: MbFormBuilder, remote: true do |f| %>
```

Comment:
Using `form_with` didn't seem to AJAX-ify the form, but passing `remote: true` to `form_for` worked correctly.